### PR TITLE
i18n - Zulu support

### PR DESCRIPTION
**Pull request by codingis4noobs2**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature to add i18n - Zulu Language Support
- **Why was this change needed?** (You can also link to an open issue here)
Fixes issue 🚀 Feature: i18n - Zulu Language Support #1592 
- **Other information**:

### CODE REVIEW

**Comment by codingis4noobs2** [comment]
@davidsoderberg @ainouzgali Review it 😀

File: docs/docs/notification-center/react-components.md [hunk 1]
```diff
@@ -212,7 +212,7 @@ The `i18n` prop can accept 2 different types of values
           <li><code>tr</code> (Turkish)</li>
           <li><code>te</code> (Telugu)</li>
           <li><code>vi</code> (Vietnamese)</li>
-          
+          <li><code>zo</code> (Zulu)</li>
```
Comment (p-fernandez) [inline docs/docs/notification-center/react-components.md:215]: Zulu ISO code is `zu`.

File: packages/notification-center/src/i18n/languages/zu.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const ZU: ITranslationEntry = {
+  translations: {
+    notifications: 'Izaziso',
```
Comment (p-fernandez) [inline packages/notification-center/src/i18n/languages/zu.ts:5]: This translation doesn't seem accurate based on reversing Google Translate.
Are you a native speaker of zulu language?

**Review by p-fernandez — CHANGES_REQUESTED** [review]

File: docs/docs/notification-center/react-components.md (hunk 1 above)
Comment (codingis4noobs2) [reply docs/docs/notification-center/react-components.md:215]: I'll fix it

Comment (codingis4noobs2) [reply docs/docs/notification-center/react-components.md]: And talking about "Notifications" , In Zulu they call it isaziso (Notice) @p-fernandez

**Review by scopsy — APPROVED** [review]

**Pull request merged**